src/transformers/file_utils.py

```diff
@@ -190,6 +190,19 @@ def is_faiss_available():
     return _faiss_available
 
 
+def require_torch(fn):
```

Comment:
Nitpick: Is `require_torch` the best name for that method, as it already exists in `testing_utils` but works slightly differently?

I fear this would make autocompletion more ambiguous